docs/streams/kafka_on_gdn/kafka-python-client.md

```diff
@@ -0,0 +1,75 @@
+---
+sidebar_position: 20
+title: Kafka Python Client
+---
+
+Kafka-Python client library is designed to function much like the official java client, with a sprinkling of pythonic interfaces(e.g., consumer iterators). The library consists 6 different APIS via classes KafkaConsumer, KafkaProducer, KafkaAdminClient, KafkaClient, BrokerConnection, ClusterMetadata.
```

Comment:
```suggestion
The Kafka-Python client library is designed to function much like the official Java client, with a sprinkling of Pythonic interfaces(e.g., consumer iterators). The library consists six different APIs via classes KafkaConsumer, KafkaProducer, KafkaAdminClient, KafkaClient, BrokerConnection, and ClusterMetadata.
```

Reply:
Is the list of the APIs important for this topic? It seems important for the library, but is it necessary information for us explaining how to use the client with Macrometa? If not, then remove it.